app/javascript/packs/components/Punches.js

```diff
@@ -2,11 +2,21 @@ import React from 'react';
 import Punch from './Punch';
 
 const Punches = ({ sheet }) => {
+
+  const worked_hours = (sheet) => {
```

Comment:
If it doesn't require a lot of setup, would you mind adding some tests for this component?